client/src/components/common/link-row.vue

```diff
@@ -12,7 +12,9 @@
     <a id="name-build-link"
        class="middle-link pt-2"
        :class="{ 'no-selector-link' : !entitySelectorUrl }"
-       href="#name-build-info"
+       href="#"
+       onclick="return false;"
```

Comment:
I think you can also do `onclick=""`.

(Or onclick.prevent or onclick.stop.)

Try it.

Reply:
Kk, will do.

Reply:
I'm not entirely sure why, but neither had the same effect.
From what i read, it was one of two options that were recommended. 
`onclick="return false;"` or `href="javascript:void(0)"`

Reply:
OK, thanks for checking.